### PR TITLE
PXB-2516 : debug build fails on Xenial-32

### PR DESCRIPTION
**Pull request by kennt-percona**
https://jira.percona.com/browse/PXB-2516

Fixing build failures.

### CODE REVIEW

**Comment by kennt-percona** [comment]
https://pxb.cd.percona.com/view/PXB%208.0/job/percona-xtrabackup-8.0-param/251/

File: storage/innobase/xtrabackup/src/quicklz/quicklz.h [hunk 1]
```diff
@@ -62,7 +62,7 @@ typedef unsigned short int ui16;
 
 // Detect if pointer size is 64-bit. It's not fatal if some 64-bit target is not
 // detected because this is only for adding an optional 64-bit optimization.
-#if defined _LP64 || defined __LP64__ || defined __64BIT__ || _ADDR64 || \
+#if defined _LP64 || defined __LP64__ || defined __64BIT__ || defined _ADDR64 || \
```
Comment (rahulmalik87) [inline storage/innobase/xtrabackup/src/quicklz/quicklz.h:65]: I see same file exists in 2.4? is it impacting build failure for 2.4 as well?

Comment (kennt-percona) [reply storage/innobase/xtrabackup/src/quicklz/quicklz.h:65]: Interestingly, the compile succeeds on 2.4 (no warnings)

Comment (rahulmalik87) [reply storage/innobase/xtrabackup/src/quicklz/quicklz.h:65]: may be some dependency version. I think better to add fix in 2.4 as well

**Comment by kennt-percona** [comment]
https://pxb.cd.percona.com/view/PXB%208.0/job/percona-xtrabackup-8.0-param/252/

File: storage/innobase/xtrabackup/src/quicklz/quicklz.h (hunk 1 above)
Comment (kennt-percona) [reply storage/innobase/xtrabackup/src/quicklz/quicklz.h:65]: Fixed in 2.4 also.  https://github.com/percona/percona-xtrabackup/pull/1113

**Review by rahulmalik87 — APPROVED** [review]
LGTM

**Pull request merged**